src/field/pokemon.ts

```diff
@@ -1955,7 +1955,7 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
   * @param {PokemonMove} battlerMove The move being used
   * @returns {HitResult} The result of the attack
   */
-  apply(source: Pokemon, move: Move): HitResult {
+  apply(source: Pokemon, move: Move, targets: number): HitResult {
```

Comment:
```suggestion
  apply(source: Pokemon, move: Move): HitResult {
```
If you end up using my other suggestion...